### ISSUE TITLE
Some questions about face recognition poisoning attack

### ISSUE THREAD

**Issue by Yeah21** [HanxunH/Unlearnable-Examples#14]
I am so interested in your poisoning attack paper that I regret not reading this article earlier. But I am confused by the face recognition poisoning attack. "100 identities from CelebA as the small dataset are used to generate the unlearnable images for 50  identities of WebFace" in your paper. I think that generates 100 perturbation noise images, and how do put them on the 50 identities? Why not generate 50 noise images by themselves just like CIFAR10 poisoning attack? This is my confusion, thank you for your answer.

**Comment by HanxunH** [comment]
Hi,

We used 50 identities of WebFace as users who want their images protected, to generate the noise, we used additional 100 identities from CelebA, so there is a total of 150 identities. 

I'm very interested to see how it goes with generating noises with only 50 identities from WebFace, my conjecture is it works the same. Please keep me updated if you have any results on this.

Best,

**Comment by Yeah21** [comment]
I am sorry to misunderstand the experimental settings. A small question is why use additional 100 identities from CelebA to generate the noise? I want to do a reproduction of this paper, so more details are better, think you again.

**Comment by HanxunH** [comment]
Considering that just generating the noise using a binary classifier, might not work well on classifiers with multiple identities. But I think it could also work with just these 50 identities without the CelebA subset.

**Comment by Yeah21** [comment]
I'm sorry to bother you again. I had an overfitting problem when reproducing the normal face classification network using your code train_clean.sh. I divided the data set of WebFace-CASIA112 into 80% training set and 20% test set. On the test set, the classification accuracy was very low, about 12%, while on the training set, the accuracy was 90%.  To find out the answer to this question, I studied insightface, a facial recognition library. But it used face verification rather than classification accuracy in the verification process. I am confused that the model has a very serious overfitting problem in the face classification task. Whether you have encountered it and how to solve it. Thank you again!

**Comment by HanxunH** [comment]
I do not have the logs for the 100% clean setting anymore, and I do not recall encountering overfitting. 
I have results for the experiment with 50 identities (unlearnable) and the rest (10525 identities) as clean. The validation classification accuracy is ~85% and the loss is ~1.11. Training ACC and loss are 99% and ~0.11. For 100% clean, results should be very similar.
